src/app/core/data/pdf-attestation-download.service.ts

```diff
@@ -0,0 +1,48 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment';
+import { Observable, catchError, of } from 'rxjs';
+import { TranslateService } from '@ngx-translate/core';
+import { NotificationsService } from 'src/app/shared/notifications/notifications.service';
+
+@Injectable()
+export class PdfAttestationDownloadService {
+
+    // Injecting httpClient into this class
+    constructor(private httpClient: HttpClient,
+        protected notificationsService: NotificationsService,
+        protected translationService: TranslateService
+        ){}
```

Comment:
Why `protected` and not just `private` ?
Maybe a little nit more indentation could be great : 
```typescript
constructor(
    private httpClient: HttpClient,
    private notificationService: NotificationService,
    private translationService: TranslateService
){ }
```